test/e2e/kubectl.go

```diff
@@ -39,9 +39,29 @@ const (
 
 var _ = Describe("kubectl", func() {
 
-	updateDemoRoot := filepath.Join(root, "examples/update-demo")
-	nautilusPath := filepath.Join(updateDemoRoot, "nautilus-rc.yaml")
-	kittenPath := filepath.Join(updateDemoRoot, "kitten-rc.yaml")
+	// Constants.
+	var (
+		updateDemoRoot = filepath.Join(root, "examples/update-demo")
+		nautilusPath   = filepath.Join(updateDemoRoot, "nautilus-rc.yaml")
+		kittenPath     = filepath.Join(updateDemoRoot, "kitten-rc.yaml")
+	)
+
+	var cmd *exec.Cmd
+
+	BeforeEach(func() {
+		cmd = kubectlCmd("proxy", "--port=8011")
```

Comment:
Should make 8011 a constant to avoid typing it wrong and make it easier to change later if we need to.